modules/deployment-model-impl/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
this sloudn't be needed, use the mockito-inline dependency instead

Reply:
The mockito-inline dependency doesn't have the utilities for static methods in 4.9.0. @hyerans verify if they are in 4.11.0. Otherwise, [I think just another option](https://www.davidvlijmincx.com/posts/writing_higher_quality_tests_with_mockitos_inline_mock_maker/).

Reply:
@elrodro83 @fsgonz We tried mockito-inline first, but mockito-inline 4.11.0.still doesn't have the utilities for the static methods. If we keep this line, mockito-core 4.11.0 works fine for the tests we have.